src/models/db/queries/requests.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../index");
+
+const add = (newRequest) => {
+  return db.oneOrNone(`
+    INSERT INTO
+      timeoff_requests (username, type_of_timeoff, date_off, time_start, time_end)
+    VALUES
+      ($1, $2, $3, $4, $5)
```

Comment:
can you fix this query and use named parameter formatting - https://github.com/vitaly-t/pg-promise/wiki/Common-Mistakes#named-parameter-formatting-is-not-used-where-it-should-be